modules/ROOT/pages/static-ip-config.adoc

```diff
@@ -0,0 +1,46 @@
+:experimental:
+= Configuring FCOS to Use a Static IP Address
+By default, an FCOS instance will attempt to grab a DHCP address from the local network. However, if you need FCOS to use a static IP address, you can do so by specifying the NetworkManager configuration in the Ignition configuration file.
+
+As with any custom configuration on FCOS, you can write specific files in the xref:ign-storage.adoc[`storage` node] of the Ignition file.
+
+The following snippet shows how to assign the following to eth0:
+
+* static IP: `192.0.2.10/24`
+* gateway: `192.0.2.1`
+* DNS: `92.168.124.1;1.1.1.1;8.8.8.8`
```

Comment:
For consistencly, the first DNS address should be `192.168.124.1`

Reply:
doh! sorry for the copy-pasta error!